config/locales/vi.yml

```
@@ -63,43 +63,43 @@ vi:
     custom_filters:
       number_of_records: Đã đạt giới hạn. Số lượng tuỳ chọn lọc tối đa cho mỗi mỗi người dùng mỗi tài khoản là 50.
-      invalid_attribute: Invalid attribute key - [%{key}]. The key should be one of [%{allowed_keys}] or a custom attribute defined in the account.
-      invalid_operator: Invalid operator. The allowed operators for %{attribute_name} are [%{allowed_keys}].
-      invalid_value: Invalid value. The values provided for %{attribute_name} are invalid
+      invalid_attribute: Khóa thuộc tính không hợp lệ - [%{key}]. Chìa khóa phải là một trong [%{allowed_keys}] hoặc thuộc tính tùy chỉnh được xác định trong tài khoản.
+      invalid_operator: Toán tử không hợp lệ. Các toán tử được phép cho %{attribute_name} là [%{allowed_keys}].
+      invalid_value: Giá trị không hợp lệ. Các giá trị được cung cấp cho %{attribute_name} không hợp lệ
   reports:
     period: Thời gian báo cáo từ %{since} đến %{until}
     utc_warning: Báo cáo đã được tạo với múi giờ UTC
     agent_csv:
       agent_name: Tên tổng đài viên
-      conversations_count: Assigned conversations
-      avg_first_response_time: Avg first response time
+      conversations_count: Cuộc trò chuyện được chỉ định
+      avg_first_response_time: Thời gian phản hồi đầu tiên trung bình
       avg_resolution_time: Avg resolution time
       resolution_count: Số lượng giải quyết
-      avg_customer_waiting_time: Avg customer waiting time
+      avg_customer_waiting_time: Thời gian chờ đợi trung bình của khách hàng
     inbox_csv:
       inbox_name: Tên kênh
       inbox_type: Kiểu kênh
       conversations_count: Số hội thoại
-      avg_first_response_time: Avg first response time
-      avg_resolution_time: Avg resolution time
+      avg_first_response_time: Thời gian phản hồi đầu tiên trung bình
+      avg_resolution_time: Thời gian giải quyết trung bình
     label_csv:
       label_title: Nhãn
       conversations_count: Số hội thoại
-      avg_first_response_time: Avg first response time
-      avg_resolution_time: Avg resolution time
+      avg_first_response_time: Thời gian phản hồi đầu tiên trung bình
+      avg_resolution_time: Thời gian giải quyết trung bình
     team_csv:
       team_name: Tên nhóm
       conversations_count: Số hội thoại
-      avg_first_response_time: Avg first response time
-      avg_resolution_time: Avg resolution time
+      avg_first_response_time: Thời gian phản hồi đầu tiên trung bình
+      avg_resolution_time: Thời gian giải quyết trung bình
       resolution_count: Số lượng giải quyết
-      avg_customer_waiting_time: Avg customer waiting time
+      avg_customer_waiting_time: Thời gian chờ đợi trung bình của khách hàng
     conversation_traffic_csv:
       timezone: Múi giờ
     sla_csv:
-      conversation_id: Conversation ID
+      conversation_id: ID hội thoại
       sla_policy_breached: SLA Policy
-      assignee: Assignee
+      assignee: Người được chỉ định
```

```suggestion
      assignee: Đại lý được chỉ định
```

Using this suggestion, as the word here means presentor where as we are looking at "assignee" means the designated agent.